src/corona_hakab_model/main.py

```diff
@@ -88,6 +97,14 @@ def main():
     args, _ = parser.parse_known_args()
     set_seeds(args.seed)
 
+    if args.sub_command == 'analyze-matrix':
+        matrix_analyzer = MatrixAnalyzer(args.matrix_path)
+        matrix_analyzer.export_raw_matrices_to_csv()
+        matrix_analyzer.analyze_histograms()
+        if args.show:
```

Comment:
Why does analyze try to plot. Return a dataframe or series instead, and then let the user choose if he want to plot it or save it to a csv